libs/datasets/data_source.py

```diff
@@ -7,7 +7,7 @@
 import structlog
 from covidactnow.datapublic import common_df
 from covidactnow.datapublic.common_fields import CommonFields
-from scripts import ccd_helpers
+from libs.datasets.sources import can_scraper_helpers as ccd_helpers
```

Comment:
Are you waiting to push the can_scraper_helpers name down? Seems we can just use that everywhere

Reply:
Yes, I'm waiting for something that will take less screen space than hundreds of `can_scraper_helpers.ScraperVariable` with all those keyword args. Also trying to preserve the existing code during the transition in case there are merge conflicts to resolve etc.